.github/workflows/ubuntu.yml

```diff
@@ -16,6 +16,11 @@ jobs:
       - name: Print CPU info
         run: cat /proc/cpuinfo
 
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
Do you want to use matrix of python versions? We have a reference from torchdata. https://github.com/pytorch/data/blob/2d94ebc6e95d4bd475a98e947781e58410386a10/.github/workflows/ci.yml#L19-L28

Reply:
@ejguan we should. But the current build/test time is not not ideal so just keep the most important test for now...